Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -96,6 +98,18 @@ var DrawerLayoutAndroid = React.createClass({
      */
     drawerWidth: ReactPropTypes.number,
     /**
+     * Specifies the lock mode of the drawer. The drawer can be locked in 3 states:
+     * - unlocked (default), meaning that the drawer will respond (open/close) to touch gestures.
+     * - locked closed, meaning that the drawer will stay closed and not respond to gestures.
+     * - locked open, meaning that the drawer will stay opened and not respond to gestures.
+     * The drawer may still be opened and closed programmatically (`openDrawer`/`closeDrawer`).
+     */
+    drawerLockMode: ReactPropTypes.oneOf([
+      DrawerConsts.DrawerLockMode.Unlocked,
+      DrawerConsts.DrawerLockMode.LockedClosed,
+      DrawerConsts.DrawerLockMode.LockedOpen
+    ]),
```

Comment:
@nicklockwood was suggesting not to use constants. These could be just strings, e.g. - "open", "closed", "unlocked".